packages/fether-react/package.json

```diff
@@ -71,8 +72,17 @@
     "rxjs": "^6.2.0"
   },
   "devDependencies": {
+    "@babel/cli": "^7.2.3",
+    "@babel/core": "^7.2.2",
+    "@babel/plugin-transform-modules-commonjs": "^7.2.0",
+    "@babel/plugin-transform-runtime": "^7.2.0",
+    "@babel/plugin-proposal-pipeline-operator": "^7.3.0",
```

Comment:
Remove all babel plugins, except plugin-proposal-decorators, which is the only one we use (configured in craco.config.js)